tests/test_multi_standbys.py

```diff
@@ -111,20 +111,21 @@ def test_005_number_sync_standbys():
     print("set number_sync_standbys = 2")
     assert node1.set_number_sync_standbys(2)
     assert node1.get_number_sync_standbys() == 2
-    print("synchronous_standby_names = '%s'" %
-          node1.get_synchronous_standby_names())
+    node1.print_synchronous_standby_names()
+    assert node1.get_synchronous_standby_names() == node1.get_synchronous_standby_names_local()
```

Comment:
It's best to use `eq_` from the nosetest package, like we do in other places. In case of failures it gives more details about what's wrong, making it easier to debug.